open-api/rest-catalog-open-api.yaml

```diff
@@ -1860,6 +2320,26 @@ components:
               items:
                 type: string
 
+    AddViewVersionUpdate:
+      allOf:
+        - $ref: '#/components/schemas/BaseUpdate'
+        - type: object
+          required:
+            - view-version
+          properties:
+            view-version:
+              $ref: '#/components/schemas/ViewVersion'
+
+    SetCurrentViewVersionUpdate:
+      allOf:
+        - $ref: '#/components/schemas/BaseUpdate'
+        - type: object
+          required:
+            - view-version-id
```

Comment:
This also needs to support -1 to signal that the version should be the last added version in the request.

Reply:
agreed, I've added a description to indicate that